client/app/hosting/database/dump/hosting-database-dump.controller.js

```diff
@@ -24,52 +22,37 @@ angular.module("App").controller(
         }
 
         loadDumps () {
-            this.dumpsLoading = true;
-            this.dumpsIds = null;
+            this.databaseDumps = undefined;
```

Comment:
`undefined` is already the default value, you don't have to write this

Reply:
this one ensure datagrid is loading while refreshing